core/audits/network-requests.js

```diff
@@ -73,6 +73,7 @@ class NetworkRequests extends Audit {
         statusCode: record.statusCode,
         mimeType: record.mimeType,
         resourceType: record.resourceType,
+        priority: record.priority,
```

Comment:
> FYI, this is the "final" priority of the resource. Is this what you need?

Yes, that's all that really matters in this case (to really reconstruct the whole load you'll need something like the full devtoolsLog anyways, not just a summary like this).

`finalPriority` seems OK but if it's compelling enough to change we should switch it on the real network-request and this can be updated to match, rather than going activist in the debug audit :)

Reply:
iirc, devtools sdk uses `initialPriority` but doesn't prefix stuff w/`final`

(yes im very aware that i heavily advocated for `final` with our lhr URL names. :)